task-requests/script.js

```diff
@@ -143,6 +184,18 @@ filterButton.addEventListener('click', (event) => {
   backDrop.style.display = 'flex';
 });
 
+function manipulateURLQueryParams(constructedQueryString) {
+  const currentURLInstance = new URL(window.location.href);
+  currentURLInstance.search = '';
+  const currentURL = currentURLInstance.href;
+  const newURLWithQueryParams = `${currentURL}${constructedQueryString}`;
+  window.history.pushState(
+    { path: newURLWithQueryParams },
+    '',
+    newURLWithQueryParams,
+  );
+}
+
```

Comment:
If there are more query other than these sort and filter queries, they gets trimmed. For example, feature-flag: dev=true will get trimmed.